pkg/config/config_test.go

```diff
@@ -29,3 +30,25 @@ func TestSetupStatsd(t *testing.T) {
 	})
 	Config.StatsdEnabled = false
 }
+
+func TestSetupPrometheus(t *testing.T) {
+	prometheus.DefaultRegisterer = prometheus.NewRegistry()
+	Config.PrometheusEnabled = false
+	setupPrometheus()
+	assert.Nil(t, Global.Prometheus.EvalCounter)
+	Config.PrometheusEnabled = true
+	setupPrometheus()
+	assert.NotNil(t, Global.Prometheus.EvalCounter)
+	assert.NotNil(t, Global.Prometheus.RequestCounter)
+	assert.Nil(t, Global.Prometheus.RequestHistogram)
+}
+
+func TestSetupPrometheusWithLatencies(t *testing.T) {
+	prometheus.DefaultRegisterer = prometheus.NewRegistry()
+	Config.PrometheusEnabled = true
```

Comment:
Can you reset the configuration after the test?

Reply:
sure, no problem